src/test/java/org/opentripplanner/transit/raptor/_data/transit/TestTransfer.java

```diff
@@ -41,6 +66,10 @@ public static TestTransfer walk(int stop, int durationInSeconds) {
         return new TestTransfer(stop, durationInSeconds, DEFAULT_NUMBER_OF_LEGS, STOP_REACHED_ON_FOOT);
     }
 
+    public static TestTransfer walk(int stop, int durationInSeconds, int opening, int closing) {
```

Comment:
Add JavaDoc here so it is clear what the opening and closing is and what the unit is.

Reply:
:+1:, I've added javadoc.